tests/stories/2fa/test_enable_2fa.yml

```diff
@@ -0,0 +1,30 @@
+version: "3.1"
+
+stories:
+  - story: User wants to enable 2fa authorization - 1
+    steps:
+      - user: |
```

Comment:
For the tests, we stopped putting the message in there because its so error prone.

so change each to

steps:
  - intent: intent_enable_2fa...